packages/backend/src/tokens/keys.ts

```diff
@@ -202,6 +208,31 @@ async function fetchJWKSFromBAPI(apiUrl: string, key: string, apiVersion: string
   });
 
   if (!response.ok) {
+    const errors = await response.json().then(({ errors }) => {
+      if (!errors) {
+        return false;
+      }
+
+      const invalidSecretKey = getErrorObjectByCode(errors, TokenVerificationErrorCode.InvalidSecretKey);
+
+      if (!invalidSecretKey) {
+        return false;
+      }
```

Comment:
❓ Can we make it a little more consistent here? I'd prefer to return `null` instead of `false` when there is no value. Is there a particular reason we return a boolean value?

Reply:
@anagstef no particular reason, I'll change it